integrations/java/workload-executor

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o errexit
+
+export JAVA_HOME="/opt/java/jdk11"
+
+OUTPUT_DIRECTORY=$PWD
+
+cd mongo-java-driver || exit
```

Comment:
Can you explain why this is needed?

Reply:
Can this be changed as follows:

```
cd mongo-java-driver && ./gradlew --info  -Dorg.mongodb.workload.output.directory="$OUTPUT_DIRECTORY" -Dorg.mongodb.test.uri="$1" -Dorg.mongodb.workload.spec="$2" driver-workload-executor:run
```

Reply:
Why is the `exit` needed?  It's just a sanity check in case the directory isn't there.